README.md

```diff
@@ -29,9 +29,14 @@ running in the Keptn ecosystem:
 |    0.12.2     |                               keptncontrib/job-executor-service:0.1.7                                |       v2       |
 |    0.12.6     |                               keptncontrib/job-executor-service:0.1.8                                |       v2       |
 |    0.13.x     |                               keptncontrib/job-executor-service:0.1.9                                |       v2       |
+|    0.13.x     |                               keptncontrib/job-executor-service:0.1.9                                |       v2       |
+|    0.14.x     |                               keptncontrib/job-executor-service:0.2.1                                |       v2       |
 
 Please note: Newer Keptn versions might be compatible, but compatibility has not been verified at the time of the release.
 
+**Note**: Versions compatible with Keptn 0.14.x and newer are not backward compatible due to a change in NATS cluster name
```

Comment:
I would add that this limitation exists only for installation in the same namespace as keptn (the default installation in a separate namespace does not have this constraint)

Reply:
You are right. I just added a comment about that.